config/application.rb

```diff
@@ -36,12 +36,12 @@ class Application < Rails::Application
     config.active_job.queue_adapter = :sidekiq
 
     # Prefix the queue name of all jobs with Rails ENV
-    config.active_job.queue_name_prefix = Rails.env
+    config.active_job.queue_name_prefix = nil
```

Comment:
As we are not doing any queue/background job-related stuff in this PR, it can be changed in https://github.com/suho/google-search-ruby/pull/45

Reply:
Fixed in [`4c447f6` (#42)](https://github.com/suho/google-search-ruby/pull/42/commits/4c447f699243eec5c046c00dd38c76964fc91c8f)